x/wasm/client/cli/new_tx.go

```diff
@@ -81,7 +81,7 @@ func UpdateContractAdminCmd(cdc *codec.Codec) *cobra.Command {
 				}
 			}
 
-			msg := types.MsgUpdateAdministrator{
+			msg := types.MsgUpdateAdmin{
```

Comment:
The rest of this function needs some updates as well. E.g.
- text "Set new admin for a contract. Can be empty to prevent further migrations"
- Block starting with `// safety net to not accidentally clear an admin`

Reply:
Yup, and I added a clear admin cmd. Forgot about this cli stuff...